source/opt/function.cpp

```diff
@@ -227,6 +227,18 @@ BasicBlock* Function::InsertBasicBlockBefore(
   return nullptr;
 }
 
+bool Function::HasEarlyReturn() const {
```

Comment:
Don't forget to remove this change once you've refactored it into a separate PR or a static method.

Reply:
Done.